static/js/utils/string_utils.test.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { getCommonPrefix } from "./string_utils";
+
+test("get common prefix", () => {
+  const cases: {
+    wordList: string[];
+    wantedPrefix: string;
+  }[] = [
+    {
+      wordList: [],
+      wantedPrefix: "",
+    },
+    {
+      wordList: ["abc"],
```

Comment:
I wanted to make this function as general as possible because it's in a shared utils file. And technically if you're looking for a common prefix among a set of words, for a single word, the prefix would be that word.

In stat_var_section_input (where this function is used), the prefix is only replaced if there is more than 1 stat var in the list.

Reply:
ah, i see. that makes sense -- could you clarify in the function comment?

Reply:
done